CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+### Fixed
+
+- Fixes an issue where wildcard `BUILD`'s are invoked from a relative directory (e.g., `earthly ./my-dir/*+target`). [#3840](https://github.com/earthly/earthly/issues/3840)
```

Comment:
Isn't this related to `BUILD ./my-dir/*+target` (in the context of `earthly ./rel-path+something`), as opposed to `earthly ./my-dir/*+target`?

Reply:
Yes, thanks. My example shouldn't contain the `*`.